docs/logs/complex-objects/Program.cs

```diff
@@ -2,17 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 using Microsoft.Extensions.Logging;
+using OpenTelemetry;
 using OpenTelemetry.Logs;
 
-var loggerFactory = LoggerFactory.Create(builder =>
-{
-    builder.AddOpenTelemetry(logging =>
-    {
-        logging.AddConsoleExporter();
-    });
-});
+var sdk = OpenTelemetrySdk.Create(builder => builder
```

Comment:
I don't think entire examples should be moved to newer pattern. It maybe a good change, but definitely outside this PR.